src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,153 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    public MyLinkedList() {
+        head = null;
+        tail = null;
+        size = 0;
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (size == 0) {
+            Node<T> node = new Node<>();
```

Comment:
when creating any Node, pass all three values to the constructor

Reply:
done